server/src/main/java/io/crate/metadata/settings/session/SessionSettingRegistry.java

```diff
@@ -155,10 +171,41 @@ public Map<String, SessionSetting<?>> settings() {
     }
 
     private static String[] objectsToStringArray(Object[] objects) {
-        String[] strings = new String[objects.length];
+        ArrayList<String> argumentList = new ArrayList<>();
         for (int i = 0; i < objects.length; i++) {
-            strings[i] = DataTypes.STRING.implicitCast(objects[i]);
+            String str = DataTypes.STRING.implicitCast(objects[i]);
+            for (String element : str.split(",")) {
+                argumentList.add(element.trim());
+            }
+        }
+        return argumentList.toArray(String[]::new);
+    }
+
+    private static void validateDateStyleFrom(String... strings) {
+        String dateStyle;
+        for (String s : strings) {
+            dateStyle = s.toUpperCase(Locale.ENGLISH);
+            switch (dateStyle) {
+                // date format style
+                case "ISO":
+                    break;
```

Comment:
I think this should be `continue`, so that we can validate the rest of the "words"

Reply:
The `break` is only relevant for the `switch` block and can only match a single `dateStyle` token.
However it is iterated over all argument tokens in the input strings in the for-loop. If any of the tokens are invalid, the validation will fail. Am I missing something?

Reply:
Sorry, I should have drank my coffee first :(